generators/cmd/templates/leaf.gotmpl

```diff
@@ -96,6 +96,12 @@ var {{ $cmdvar }} = &cobra.Command{
   Short: TRAPI("{{.Short}}"),
   Long: TRAPI(`{{.Long}}`),
   RunE: func(cmd *cobra.Command, args []string) error {
+    {{- if .Deprecated}}
+    if _, err := os.Stderr.Write([]byte(fmt.Sprintf("[warn] "+TRCLI("cli.deprecated-api"), "{{.Path}}")+"\n")); err != nil {
```

Comment:
We have `printfStderr()` in apiclient.go, which can be used in the files in `cmd` package.

By the way, I found `warning:` and `WARN: ` prefixes in other files ... :(

We might need to create a function to output a warning message like `printWarning()` and consolidate to use it.